requirements-testing.txt

```diff
@@ -0,0 +1,10 @@
+coverage==4.5.4
+cython==0.29.13
+garnett==0.5.0
+GitPython==3.0.2
```

Comment:
We don't actually need GitPython for anything, do we?

Reply:
@bdice indicated that he intends to use this file for CI purposes. In anticipation of performing benchmarks on CI, we will keep GitPython in this file for now.